.github/ISSUE_TEMPLATE/design.md

```diff
@@ -0,0 +1,16 @@
+---
+name: Design
+about: Track or request design work
+title: 'Design [...] so that [...].'
+labels: design
+assignees: '@gissoo'
```

Comment:
I'd prefer not to have assignee hard-coded; Gissoo should self-assign when and if she starts working on something.